src/App.tsx

```diff
@@ -1,14 +1,62 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [status, setStatus] = useState<string>('all');
+  const [query, setQuery] = useState<string>('');
+
+  useEffect(() => {
+    getTodos()
+      .then(
+        setTodos,
+        setFilteredTodos,
+      );
+  }, []);
+
+  const filteredByStatus = (todosFromServer: Todo[], StateStatus: string) => {
+    switch (StateStatus) {
```

Comment:
Rename the function parameter 'StateStatus' to 'stateStatus' to follow camelCase naming convention.